tests/fixtures_without_default/dag_factory.yml

```diff
@@ -0,0 +1,69 @@
+default:
```

Comment:
slightly confused with the placement of this file. The directory is `fixtures_without_default` and the file contains a default block.

Reply:
It's a fixtures directory without a `defaults.yaml` file 🙈 
Would it be clearer if I renamed to `fixtures_without_default_yaml`?